src/Viewer.tsx

```diff
@@ -764,18 +769,33 @@ function Viewer(): ReactElement {
                       isFeatureCategorical ? (
                         <CategoricalColorPicker categories={featureCategories} disabled={disableUi} />
                       ) : (
-                        <LabeledSlider
-                          type="range"
-                          min={colorRampMin}
-                          max={colorRampMax}
-                          minSliderBound={featureKey !== null ? dataset?.getFeatureData(featureKey)?.min : undefined}
-                          maxSliderBound={featureKey !== null ? dataset?.getFeatureData(featureKey)?.max : undefined}
-                          onChange={function (min: number, max: number): void {
-                            setColorRampRange([min, max]);
-                          }}
-                          marks={getColorMapSliderMarks()}
-                          disabled={disableUi}
-                        />
+                        <Tooltip
+                          trigger={["hover", "focus"]}
+                          title={
+                            disableRampControlsUi
+                              ? "Color ramp adjustment is disabled when a Glasbey color map is selected."
+                              : undefined
+                          }
+                        >
+                          <div style={{ width: "100%" }}>
+                            <LabeledSlider
+                              type="range"
+                              min={colorRampMin}
+                              max={colorRampMax}
+                              minSliderBound={
+                                featureKey !== null ? dataset?.getFeatureData(featureKey)?.min : undefined
+                              }
+                              maxSliderBound={
+                                featureKey !== null ? dataset?.getFeatureData(featureKey)?.max : undefined
+                              }
+                              onChange={function (min: number, max: number): void {
+                                setColorRampRange([min, max]);
+                              }}
+                              marks={getColorMapSliderMarks()}
+                              disabled={disableUi || disableRampControlsUi}
```

Comment:
Disabled the ramp controls when glasbey is selected, since the range doesn't do anything for categorical ramps.